pkgs/development/ocaml-modules/nice_parser/default.nix

```
@@ -0,0 +1,38 @@
+{
+  lib,
+  buildDunePackage,
+  fetchFromGitHub,
+  menhir,
+  ppx_jane,
+  base,
+  stdio,
+  odoc,
+}:
+
+buildDunePackage rec {
+  pname = "nice_parser";
+  version = "1.0.0";
+  minimalOCamlVersion = "4.05";
+
+  src = fetchFromGitHub {
+    owner = "smolkaj";
+    repo = "nice-parser";
+    tag = version;
+    hash = "sha256-h1rqdv19tUH3CsL3OLsTmKir7YCYt4PaW19ymotEvIY=";
+  };
+
+  propagatedBuildInputs = [
+    menhir
+    ppx_jane
+    base
+    stdio
+    odoc
```

```suggestion
    stdio
```

The other libraries are not needed (see for instance https://raw.githubusercontent.com/smolkaj/nice-parser/refs/tags/1.0.0/src/dune).

---

Oh, indeed! I was being misguided by the [opam file](https://raw.githubusercontent.com/smolkaj/nice-parser/refs/tags/1.0.0/nice_parser.opam).